subcommands/keys/tuf_utils.go

```diff
@@ -354,6 +354,7 @@ func removeUnusedTufKeys(root *client.AtsTufRoot) {
 		if !found {
 			fmt.Println("= Removing unused key:", k)
 			delete(root.Signed.Keys, k)
+			delete(root.Signed.KeyOwners, k)
```

Comment:
iiuc, the key ID == owner ID (`root.Signed.KeyOwners[key.signer.Id]`)?

Reply:
It is the key-id to owner-id map.
Owner ID is a user ID (of a user who created a key).

Reply:
I see that `key.signer.Id` is used as a key in the `root.Signed.KeyOwners` map. It makes me think that a signer ID is equal to a key ID.

Reply:
https://github.com/foundriesio/fioctl/blob/eb5d7c91d08e3c9e8c7be4eb042c85e1b906aff3/subcommands/keys/tuf_utils.go#L333

Reply:
Signer here is a Golang's crypto-signer interface, not a person who owns a signing key.